components/dashboard/ActivityInputDialog.js

```diff
@@ -290,6 +292,21 @@ function ActivityInputDialog({ show, onClose }) {
                 ))}
               </Select>
             </FormControl>
+            <FormControl className={classes.formControl}>
+              <InputLabel shrink htmlFor="briefDescription-textfield">
+                Brief Description
```

Comment:
Could you please confirm with @endely / Alex C. that we want to go with "Description" and "Brief Description"? Feels… very description'y.

Reply:
I believe it's okay with them